lib/matplotlib/tri/tricontour.py

```diff
@@ -274,6 +258,22 @@ def tricontour(ax, *args, **kwargs):
     """
     %(_tricontour_doc)s
 
+    corner_mask : bool, default: :rc:`contour.corner_mask`
+        Enable/disable corner masking, which only has an effect if *Z* is
+        a masked array.  If ``False``, any quad touching a masked point is
+        masked out.  If ``True``, only the triangular corners of quads
+        nearest those points are always masked out, other triangular
+        corners comprising three unmasked points are contoured as usual.
+
+    nchunk : int >= 0, optional
+        If 0, no subdivision of the domain.  Specify a positive integer to
+        divide the domain into subdomains of *nchunk* by *nchunk* quads.
+        Chunking reduces the maximum length of polygons generated by the
+        contouring algorithm which reduces the rendering workload passed
+        on to the backend and also requires slightly less RAM.  It can
+        however introduce rendering artifacts at chunk boundaries depending
+        on the backend, the *antialiased* flag and value of *alpha*.
+
```

Comment:
`corner_mask` and `nchunk` aren't available in either `tricontour` or `tricontourf`, so should be completely removed from this file.  Sorry I wasn't cleared about this earlier.

Normally I would say that the list of valid arguments should be obvious from the Python function definitions.  But in this case, because of the class hierarchy of `TriContourSet` and `ContourSet`, etc, it isn't really clear at all.  But it is certainly true that the C++ code that ultimately calculates the tri contours doesn't know anything about chunks or corner masks.

Reply:
I have made the changes. Sorry for the constant errors. I hope everything is correct now.